lib/pbench/server/api/resources/users_api.py

```diff
@@ -308,13 +307,12 @@ def __init__(self, logger, auth):
 
     def get_valid_target_user(
         self, target_username: str, request_type: str
-    ) -> "TargetUser":
+    ) -> TargetUser:
```

Comment:
You said flake8 complained about this; but does that mean you didn't run flake8 locally before pushing the commit, or was your local flake8 happy but the Jenkins container flake8 complained?

Reply:
yeah I didn't think it would be a flake8 issue and pushed the changes without running locally, just to realize it actually is an issue.

Reply:
Yeah, I've done the same thing with a "trivial" last-minute tweak. I was just checking whether we had a flakey flake8! ;-)